extensions/iconify/src/search-icons.tsx

```diff
@@ -0,0 +1,50 @@
+import { Action, ActionPanel, Color, List } from '@raycast/api';
+import { useState } from 'react';
+
+import Service, { Icon } from './service';
+import { toBase64, toSvg } from './utils';
+
+const service = new Service();
+
+function Command() {
+  const [icons, setIcons] = useState<Icon[]>([]);
+  const [isLoading, setLoading] = useState(false);
+
+  async function queryIcons(text: string) {
+    setLoading(true);
+    setIcons([]);
+    const icons = await service.queryIcons(text);
+    setIcons(icons);
+    setLoading(false);
+  }
+
+  return (
+    <List throttle isLoading={isLoading} onSearchTextChange={queryIcons}>
+      {icons.map((icon) => {
```

Comment:
Mind adding an [EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) for when there are no items to show? It's a bit nicer than greeting users with a `No Results` page 😄

Reply:
👍 Good one, will add in a bit.

Reply:
Added in [41687ff](https://github.com/raycast/extensions/pull/1350/commits/41687ff37950d0771ba70c46b85881f6ef7bf537)